datastore/adapter/keytransform.py

```diff
@@ -17,11 +18,14 @@
 	
 	"BinaryNestedPathAdapter",
 	"ObjectNestedPathAdapter",
+	
+	"BinaryFlatFSAdapter",
+	"ObjectFlatFSAdapter"
 )
 
 
-
 KEY_TRANSFORM_T = typing.Callable[[datastore.Key], datastore.Key]
+T = typing.TypeVar('T', bound="_Adapter")
```

Comment:
The bound here should be on `"_FlatFSAdapter"`, not just `"_Adapter"`. That may also fix many mypy complains, if I'm not mistaken…